src/sipp.cpp

```diff
@@ -2271,10 +2267,4 @@ int main(int argc, char *argv[])
     close(epollfd);
     free(epollevents);
 #endif
-
-    if (scenario_file != NULL) {
-        delete [] scenario_file;
-        scenario_file = NULL;
-    }
-
```

Comment:
I do care about valgrind. If I want to find leaks, it doesn't help that we're leaking on purpose. Even though it's slightly faster to let just the OS clean it up, I find it poor coding practise. (In fact, I did the exact opposite in the fix/valgrind branch 6076caa.)

- Maintaining a suppression file is about as much work as simply cleaning up your memory.
- A suppression file probably only works for valgrind and not for other modes of memory leak detection.
- Much of the stuff *is* cleaned up right now. If we decide to ignore some, that's just sloppy. There should at least be some way to identify that a particular variable or set of variables won't be cleaned up.